io.github.cgueret.Scriptorium.metainfo.xml

```diff
@@ -0,0 +1,106 @@
+        <li>Use the preview panel to see and adjust the look of the final book</li>
+        <li>Versioning managed via Git on the back-end using YAML and HTML files</li>
+      </ul>
+    </p>
+  </description>
+
+  <developer id="tld.vendor">
+    <name>Christophe Gueret</name>
+  </developer>
+
+  <!-- Requered: Should be a link to the upstream homepage for the component -->
+  <url type="homepage">https://github.com/cgueret/Scriptorium</url>
+  <!-- Recommended: It is highly recommended for open-source projects to display the source code repository -->
+  <url type="vcs-browser">https://github.com/cgueret/Scriptorium</url>
+  <!-- Should point to the software's bug tracking system, for users to report new bugs -->
+  <url type="bugtracker">https://github.com/cgueret/Scriptorium/issues</url>
+  <!-- Should link a FAQ page for this software, to answer some of the most-asked questions in detail -->
+  <!-- URLs of this type should point to a webpage where users can submit or modify translations of the upstream project -->
+  <!--
+    <url type="translate">https://example.org/translate</url>
+    <url type="faq">https://example.org/faq</url>
+  -->
+  <!--Sould provide a web link to an online user's reference, a software manual or help page -->
+  <!--<url type="help">https://example.org/help</url>-->
+  <!-- URLs of this type should point to a webpage showing information on how to donate to the described software project -->
+  <url type="donation">https://buymeacoffee.com/cgueret</url>
+  <!--This could for example be an HTTPS URL to an online form or a page describing how to contact the developer -->
+  <url type="contact">https://mastodon.green/@christophegueret</url>
+  <!--URLs of this type should point to a webpage showing information on how to contribute to the described software project -->
+  <!--<url type="contribute">https://example.org/contribute</url>-->
+
+  <translation type="gettext">scriptorium</translation>
+  <!-- All graphical applications having a desktop file must have this tag in the MetaInfo.
+     If this is present, appstreamcli compose will pull icons, keywords and categories from the desktop file. -->
+  <launchable type="desktop-id">io.github.cgueret.Scriptorium.desktop</launchable>
+  <!-- Use the OARS website (https://hughsie.github.io/oars/generate.html) to generate these and make sure to use oars-1.1 -->
+  <content_rating type="oars-1.1"/>
+
+  <!-- Applications should set a brand color in both light and dark variants like so -->
+  <branding>
+    <!-- Two different shades of brown to refer to the color of a scriptorium desk -->
+    <color type="primary" scheme_preference="light">#865e3c</color>
+    <color type="primary" scheme_preference="dark">#cdab8f</color>
+  </branding>
+
+  <screenshots>
+    <screenshot type="default">
+      <image>https://raw.githubusercontent.com/cgueret/Scriptorium/refs/heads/main/data/screenshots/main_window.png</image>
```

Comment:
never reference the tip of a branch for screenshots.

(this applies to all)

Reply:
Okay, I'll try to figure out a better link

Reply:
Switched to a specific commit